generate-client.sh

```diff
@@ -1 +1,5 @@
-./vendor/k8s.io/code-generator/generate-groups.sh all github.com/hobbyfarm/gargantua/pkg/client github.com/hobbyfarm/gargantua/pkg/apis "hobbyfarm.io:v1 terraformcontroller.cattle.io:v1"
+./vendor/k8s.io/code-generator/generate-groups.sh all \
+    github.com/hobbyfarm/gargantua/pkg/client \
+    github.com/hobbyfarm/gargantua/pkg/apis \
+    "hobbyfarm.io:v1 terraformcontroller.cattle.io:v1" \
+    --go-header-file ./hack/boilerplate.go.txt
```

Comment:
`hack/boilerplate.go.txt` didn't make it to your PR. Without it, all of the files under `pkg/client` lose their license text.

Reply:
good catch